src/AvaTax.TaxModule.Core/ModuleConstants.cs

```diff
@@ -0,0 +1,113 @@
+using System.Collections.Generic;
+using VirtoCommerce.Platform.Core.Settings;
+
+namespace AvaTax.TaxModule.Core
+{
+    public static class ModuleConstants
+    {
+        public static class Settings
+        {
+            public static class Avalara
+            {
+                public static SettingDescriptor AvalaraTaxEnabled = new SettingDescriptor
+                {
+                    Name = "Avalara.IsEnabled",
+                    GroupName = "Taxes|Avalara",
+                    ValueType = SettingValueType.Boolean,
+                };
+
+                public static SettingDescriptor AccountNumber = new SettingDescriptor
+                {
+                    Name = "Avalara.Credentials.AccountNumber",
+                    GroupName = "Taxes|Avalara",
+                    ValueType = SettingValueType.Integer,
+                };
+
+                public static SettingDescriptor LicenseKey = new SettingDescriptor
+                {
+                    Name = "Avalara.Credentials.LicenseKey",
+                    GroupName = "Taxes|Avalara",
+                    ValueType = SettingValueType.SecureString,
+                };
+                public static SettingDescriptor CompanyCode = new SettingDescriptor
+                {
+                    Name = "Avalara.Credentials.CompanyCode",
+                    GroupName = "Taxes|Avalara",
+                    ValueType = SettingValueType.ShortText,
+                };
+                public static SettingDescriptor ServiceUrl = new SettingDescriptor
+                {
+                    Name = "Avalara.Credentials.ServiceUrl",
+                    GroupName = "Taxes|Avalara",
+                    ValueType = SettingValueType.ShortText,
+                };
+                public static SettingDescriptor AdminAreaUrl = new SettingDescriptor
+                {
+                    Name = "Avalara.Credentials.AdminAreaUrl",
+                    GroupName = "Taxes|Avalara",
+                    ValueType = SettingValueType.ShortText,
+                };
+                public static SettingDescriptor SynchronizationCronExpression = new SettingDescriptor
+                {
+                    Name = "Avalara.ScheduledOrdersSynchronization.CronExpression",
+                    GroupName = "Taxes|Avalara",
+                    ValueType = SettingValueType.ShortText,
+                    DefaultValue = "0 0 * * *"
+                };
+
+                public static IEnumerable<SettingDescriptor> AllSettings
+                {
+                    get
+                    {
+                        return new List<SettingDescriptor>
```

Comment:
Please divide settings into 2 different classes:
For avalara connection and synchronization

Reply:
done